src/components/Icon/Icon.jsx

```diff
@@ -60,6 +61,11 @@ const Icon = createClass({
 			isClickable to be false.
 		`,
 
+		onClick: func`
```

Comment:
Since onClick is a function and not a propType, does this bit need to be here?

Reply:
It probably wasn't super clear from my ticket description, but this should actually be `onSelect`. We want the component _users_ to start using `onSelect` rather than `onClick`. In fact, many of them are probably already using `onClick` today (because of the way we `passThrough` unknown props) and we have to be careful not to break their existing usage of `onClick`.

Reply:
callback functions are also defined as propTypes so this is correct